doc/mgr/telemetry.rst

```diff
@@ -337,13 +338,13 @@ To participate in a leaderboard in the `public dashboards
    ceph config set mgr mgr/telemetry/leaderboard true
 
 The leaderboard displays basic information about the cluster. This includes the
-total storage capacity and the number of OSDs. To add a description of the
-cluster, run a command of the following form: 
+cluster's total storage capacity and the number of OSDs. To add a description
+of the cluster so that it can more easily be identified on the leaderboard, run
+a command of the following form: 
 
 .. prompt:: bash #
 
```

Comment:
It's late and I'm punchy, so I will suggest changing the below to 'Ceph cluster for Computational Demonology at Miskatonic University"